zebra-network/src/isolated/tor.rs

```diff
@@ -45,6 +45,41 @@ pub async fn connect_isolated_tor(
     hostname: String,
     user_agent: String,
 ) -> Result<BoxService<Request, Response, BoxError>, BoxError> {
+    let tor_stream = new_tor_stream(hostname).await?;
+
+    // Calling connect_isolated_tor_with_inbound causes lifetime issues
+    connect_isolated(network, tor_stream, user_agent).await
+}
+
+/// Creates an isolated Zcash peer connection to `hostname` via Tor.
+/// This method is for testing purposes only.
+///
+/// See [`connect_isolated_with_inbound`] and [`connect_isolated_tor`] for details.
+///
+/// # Privacy
+///
+/// This method can make the isolated connection send different responses to peers,
```

Comment:
Minor: `method` -> `connection`

```suggestion
/// Creates an isolated Zcash peer connection to `hostname` via Tor.
/// This function is for testing purposes only.
///
/// See [`connect_isolated_with_inbound`] and [`connect_isolated_tor`] for details.
///
/// # Privacy
///
/// This function can make the isolated connection send different responses to peers,
```

Reply:
Fixed in 276fb976e